datatypes/src/raster/grid_dimension.rs

```diff
@@ -111,9 +187,11 @@ impl<I> Dim<I> {
     pub fn new(dimension_size: I) -> Dim<I> {
         Dim { dimension_size }
     }
+    #[inline]
```

Comment:
Is there a reason why you start writing `#[inline]` to all functions?

Reply:
i guess with LTO it is not required anymore.

Reply:
removed them